lib/translations.json

```diff
@@ -998,10 +998,6 @@
       "key": "publish/preview/desktop",
       "value": "Desktop"
     },
-    {
-      "key": "publish/preview/404",
-      "value": "Dokument oder Schema nicht verfügbar."
-    },
```

Comment:
this was missing in the google sheet, I've added it, please rerun `npm run translations`